decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -51,11 +51,12 @@ class Comment extends Component {
             { body }
           </p>
         </div>
-        {this._renderReplies()}
         <div className="comment__footer">
```

Comment:
Yep, with the latest design change, the replies will render bellow de reply button, so it's more clear to the user knowing which comment/reply is answering.